airflow/dags/gtfs_views_staging/stops_clean.sql

```diff
@@ -6,7 +6,9 @@ dependencies:
 ---
 
 SELECT
-    * EXCEPT(calitp_deleted_at)
+    * EXCEPT(calitp_deleted_at, stop_lat, stop_lon)
+    , SAFE_CAST(stop_lat AS FLOAT64) as stop_lat
+    , SAFE_CAST(stop_lon AS FLOAT64) as stop_lon
```

Comment:
do we want to alert on nulls somehow?

Reply:
It seems like the validator would flag this with a fairly serious error. Not sure what kind of action we'd want to do with an alert about null values from within here that we couldn't do with a similar validator error output.

Reply:
A few points here:
* `stop_lon` is only conditionally required (if `location_type` = 3 or 4 it is not required; those are uncommon but still a naive check for nulls would have false positives.)
* This feed did throw a bunch of `invalid_row_length` notices (severity `ERROR`) in its validation results.